### DISCUSSION TITLE
刚刚在微信群看到说极客时间推出了一个minispring课程，想问下是本人吗？还是说又被侵权了……

### DISCUSSION THREAD

**Discussion by alexcdever** [DerekYRC/mini-spring#48] [General]
如题

<img width="684" alt="b0674b335b845ccfca05277e0c1515e" src="https://user-images.githubusercontent.com/14200813/225251146-5e810e9a-40ff-45d1-9a35-69c44bb33e8c.png">

**Comment by DerekYRC** [comment]
@alexcdever 谢谢这位热心的朋友，不是本人哈，是否侵权还取决于课程内容。
